### DISCUSSION TITLE
What's new page doesn't show the latest version on desktop

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#13583] [Q&A] [answered]
After yesterday's release, the page at https://ravendb.net/docs/article-page/5.3/csharp/start/whats-new doesn't contain the new version when viewed on desktop. I tried it on two different computers in three different browsers just now (same internet connection, though), and none of the desktop browsers show the new version. I reloaded in every imaginable way and also used "clean" browsers etc.

When downloading the page using wget or curl, the latest version is still not present (curl log file attached: [whatsnew.zip](https://github.com/ravendb/ravendb/files/7984333/whatsnew.zip)).

However, at the same time, two different android devices show the latest version (using the same internet connection as the desktop browsers).

This seems like it's not a client-side issue but rather a difference in server-side handling or caching for mobile vs. other clients.

Any ideas?

**Comment by mateuszbartosik** [comment, marked as answer]
@krivulcik thank you for your report. We're currently investigating the issue. I will let you know once it's fixed.

**Reply by krivulcik** [reply]
Additional information which might be helpful - I have noticed that the What's new page didn't show the latest version immediately after release even with the past versions. It usually appeared with a delay (at least several hours). I didn't check on mobile in the past, so I'm unsure whether it was available for mobile clients.

**Reply by mateuszbartosik** [reply]
@krivulcik It should be working properly now. Can you confirm?

**Reply by krivulcik** [reply]
Yes, the latest version is shown on the page now, thanks.
